### PR TITLE
Change label for polygons color

### PR DESCRIPTION
**Pull request by rubenmoya**
Related to https://github.com/CartoDB/cartodb/issues/12768

When changing the labels from `fill` to `size/color` we didn't use the right copy for polygons, it has to be `color` instead of `size/color`

screen shot 2017-09-11 at 11 07 55

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/editor/style/style-form/style-form-components-dictionary.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ module.exports = {
       animationType: params.animationType
     });
     var isPointGeometry = params.queryGeometryModel && params.queryGeometryModel.get('simple_geom') === 'point';
+    var isPolygonGeometry = params.queryGeometryModel && params.queryGeometryModel.get('simple_geom') === 'polygon';
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/editor/style/style-form/style-form-components-dictionary.js]: we could create a helper `hasGeometryOf('polygon')` here to be reusable.

Comment (rubenmoya) [reply lib/assets/core/javascripts/cartodb3/editor/style/style-form/style-form-components-dictionary.js]: Done, but I'm not sure if I did it right

**Comment by nobuti** [comment]
🚀

**Comment by nobuti** [comment]
Retest this please

**Comment by nobuti** [comment]
Retest this please

**Comment by nobuti** [comment]
Retest this please

**Comment by nobuti** [comment]
retest this please

**Comment by nobuti** [comment]
retest this please

**Pull request merged**